src/containers/PerfherderGraphContainer/index.jsx

```diff
@@ -52,6 +72,13 @@ class PerfherderGraphContainer extends Component {
           options={options}
```

Comment:
The reason I defined `options.tooltips.custom` inside `PerfherderGraphContainer` is so I could have access to `tooltipsModel`because it's the only way we can know if the tooltip is going to show up or not. If you declare such custom function anywhere else, you'll need to link to the argument of the function `tooltipsModel` and bring it back to `PerfherderGraphContainer` somehow.

There's something else. Until now we were only modifying `perfherderFormatter` which is like `generateOptions` but specific to perfherder (last two charts at the bottom). If we modify `generateOptions` we will change the behaviour in all the charts not just the perfherder. 

In my opinion, `CustomTooltip` should be inside `ChartJsWrapper`, next to `<Chart />`, but because the original issue was only refered to the perfherder, I thought that solving this issue would be an experiment, and once it's fully implemented, we could move the code to `ChartJsWrapper`and apply it anywhere else.